test/integration/flags.go

```diff
@@ -29,3 +29,4 @@ func TestMain(m *testing.M) {
 
 var binaryPath = flag.String("binary", "../../out/minikube", "path to minikube binary")
 var args = flag.String("minikube-args", "", "Arguments to pass to minikube")
+var versioned = flag.Bool("versioned", false, "Run tests on all supported k8s versions")
```

Comment:
I think this might actually be a bit easier if you make this a "tag" instead of a flag. You'd have to split out the RunVersionedFunctional to it's own file, and add a tag like

```
// +build integration versioned
```